spec/system/organization/user_views_an_organization_spec.rb

```diff
@@ -51,6 +44,17 @@
         visit "/#{organization.slug}"
       end
     end
+
+    context "when more than 8 articles" do
+      before do
+        create_list(:article, 42, organization: organization)
```

Comment:
Minor: 42 means this will create unnecessary objects, I think 9 would be sufficient here.